cmd/panfs_test.go

```diff
@@ -399,4 +399,13 @@ func TestPANFSMakeBucket(t *testing.T) {
 	if !errors.Is(err, PanFSInvalidBucketPath{BucketPath: disk}) {
 		t.Fatalf("Expected error %v, got %v", PanFSInvalidBucketPath{BucketPath: disk}, err)
 	}
+
+	// Test make bucket for nested panfs path (ex: 1st bucket - /path/defaultDir, 2nd - /path/default - this should work)
```

Comment:
It wasn't obvious to me what this test case was for after reading this comment.
Can we reverse the order: give an example what is wrong and then explain why? Like:
> // Make sure it is possible to create `<bucketPath>/inner` if `<bucketPath>/innerDir` already exists.
> // In other words, if the path to a new bucket is a string prefix to an existing bucket, it should be allowed.
> // What is not allowed is nesting a bucket dir in an existing bucket dir.